### DISCUSSION TITLE
OLM or Helm chart?

### DISCUSSION THREAD

**Discussion by jabbrwcky** [OT-CONTAINER-KIT/redis-operator#273] [Q&A] [answered]
Hi, we had installed the Redis-Operator via OLM, but its page on [artifacthub.io](https://artifacthub.io/packages/olm/community-operators/redis-operator) is still on release 0.9.0, not the current 0.10.0

Is OLM still supported or do we have to switch to Helm?

**Comment by iamabhishek-dubey** [comment, marked as answer]
It's updated there
